webapp/src/js/bootstrapper/translator.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var TRANSLATIONS = {
+  en: {
+    LOAD_APP:      'Loading app…',
+    FETCH_INFO:    function (x) { return 'Fetching info (' + x + ' docs)…'; },
+    LOAD_RULES:    'Loading rules…',
+    ERROR_MESSAGE: 'Loading error, please check your connection.',
+    TRY_AGAIN:     'Try again',
+  }
+};
+
+function Translator(locale, translationData) {  
+  this.translationData = translationData || TRANSLATIONS;
+
+  Translator.prototype.setLocale = function setLocale(locale) {
```

Comment:
Is the difference between
```javascript
Translator.prototype.setLocale = function setLocale(locale) {
```
and
```javascript
Translator.prototype.setLocale = function(locale) {
```
that the former gives nicer stacktraces?

Reply:
Ya exactly. It is enforced in AirBnB's style guidelines to create cleaner stacks.  Just a habit, can remove if you don't like.